pages/visit.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Visiting
+permalink: /visit/
+---
+
+### Table of contents
+
+* [Location](#location)
+* [Access](#access)
+* [Conduct while in the space](#conduct-while-in-the-space)
+* [Accessibility](#accessibility)
+
+### Location
+
+We are located in Lower Potrero Hill at [1250 Missouri St. #111](https://www.google.com/maps/place/1250+Missouri+St,+San+Francisco,+CA+94107/@37.7504873,-122.3978271,17z/data=!3m1!4b1!4m2!3m1!1s0x808f7fae0730e01b:0x8ee0ca3bde3eae0d), San Francisco.
+
+**Parking a car:**
+there's some parking (one-hour until 6 pm, except on Sunday) on the street in front of the space, but this may be full. There is also street parking along Connecticut Street.
+
+**Bike parking:**
+feel free to bring your bike up the elevator and inside the space.
+
+**Public transit:**
+the 19 bus stops around the corner, and the 48 bus stops nearby. If you're coming from the East Bay, one way is to get off at Civic Center BART and take the 19 south.
+
+Here’s what our building looks like! When you turn from Cesar Chavez St. onto Missouri St., it’s the tall grey box with large windows:
+
+![Building from street]({{ "/assets/images/building-from-street.jpg" | relative_url }})
+
+The left side of the building has a little tree in front of the main entrance, usually behind a few cars:
+
+![Building closer]({{ "/assets/images/building-closer.jpg" | relative_url }})
+
+That’s the building’s front door. If you’re a guest of a member, or a guest attending a public event, you can use the keypad to press #111 (our unit number) to call the phone inside the space.
+
+### Access
+
+Other than during events open to the public, Double Union visitors must be the invited guest of a member. Please do not visit without the express invitation of a member. Please do not invite guests unless you are a member. Guests may be any gender or age.
+
+We announce public events on [our blog]({{ site.url.tumblr }}), [public mailing list]({{ site.url.mailing_list }}), [Facebook]({{ site.url.facebook }}), [Instagram]({{ site.url.instagram }}), and on Twitter at [{{ site.twitter_username }}]({{ site.url.twitter }}).
```

Comment:
These links are rendering as empty links (like `<a href="">our blog</a>`) - based on the other links, it looks like they may need to be `site.urls.*` instead of `site.url.*`